src/python/espressomd/version.pxd

```diff
@@ -0,0 +1,5 @@
+cdef extern from "version.hpp":
+    cdef int ESPRESSO_VERSION
+    cdef const char * GIT_BRANCH
+    cdef const char * GIT_COMMIT_HASH
```

Comment:
In version.hpp, these are macros and not strings. Does importing them really work like that?

Reply:
Apparently, but now that I think of it they don't need to be macros in the first place.